packages/cli/tests/unit/agentInstall/installCommand.spec.ts

```diff
@@ -799,6 +800,28 @@ appmap_dir: tmp/appmap
       sinon.stub(commandRunner, 'run').resolves({ stdout: '', stderr: '' });
     });
 
+    it("tells the user if the target directory doesn't exist", async () => {
+      expect.assertions(5);
+
+      const badDirectory = '/tmp/foo';
+      expect(existsSync(badDirectory)).toBeFalsy(); // sanity check
+
+      const log = jest.spyOn(console, 'error');
+      const installProcedureStub = sinon
```

Comment:
This mixing of sinon and jest is a bit curious.

Reply:
I just grabbed bits that I needed from other places in this file....

I switched to using `sinon` exclusively.